docs/api.rst

```diff
@@ -50,3 +52,11 @@ Pandas Utils
 .. automethod:: api.client.gro_client.GroClient.add_single_data_series
 
 .. automethod:: api.client.gro_client.GroClient.get_data_series_list
+
+*************
+Crop Modeling
+*************
+
+.. automethod:: api.client.crop_model.CropModel.compute_weights
```

Comment:
These docstrings should just have single backticks, instead of triple

Reply:
done